apps/balancer-tools/src/app/apr/pool/(components)/PoolOverviewCards.tsx

```diff
@@ -38,7 +42,11 @@ export default async function PoolOverviewCards({
         { title: "TVL", content: formatNumber(tvl) },
         { title: "APR", content: formatNumber(apr).concat("%") },
         { title: "Round Number", content: roundId },
-        { title: "Round Ended", content: formatDate(round.endDate) },
+        {
+          title: "Round Ended",
+          content: formatDate(round.endDate),
+          tooltip: "Every round ends on a Thursday at 00:00 UTC",
```

Comment:
```suggestion
          tooltip: "Rounds end every Wed at 23:59:59 UTC",
```

@mendesfabio can confirm what the definition is but I'm almost certain this is the case